packages/thirdweb/src/react/core/hooks/transaction/useSendTransaction.ts

```diff
@@ -175,10 +176,11 @@ export function useSendTransactionCore(args: {
                 chain: tx.chain,
               }),
               _erc20Value?.tokenAddress
-                ? getWalletBalance({
+                ? getTokenBalance({
                     client: tx.client,
-                    address: account.address,
```

Comment:
wait how did i miss this? pretty sure that was working before :/ thanks for fixing

Reply:
It may have been, a user is saying it broke ~v5.48